### ISSUE TITLE
Rejects multiple characterSet or progressCode

### ISSUE THREAD

**Issue by ebleys** [metadata101/iso19115-3#27]
ISO 19139 allows multiple gmd:characterSet and gmd:progressCode and spatialRepresentationCode
Current XSLt give SEVERE error if > 1 of either is used
(I simplified the XML and got a result)
tried xsl:for-each in core.xsl but didn't punch through the problem

Sorry
E

**Comment by fxprunayre** [comment]
indeed, 

```
      <spatialRepresentationType>
        <MD_SpatialRepresentationTypeCode codeListValue="grid"
          codeList="./resources/codeList.xml#MD_SpatialRepresentationTypeCode"/>
      </spatialRepresentationType>
      <spatialRepresentationType>
        <MD_SpatialRepresentationTypeCode codeListValue="vector"
                                          codeList="./resources/codeList.xml#MD_SpatialRepresentationTypeCode"/>
      </spatialRepresentationType>
```
error in fromISO19139.xsl is
```
System ID: /data/dev/gn/iso19115-3/src/main/plugin/iso19115-3/convert/ISO19139/utility/multiLingualCharacterStrings.xsl
Scenario: raster
XML file: /data/dev/gn/3.4.x/schemas/iso19139/src/main/plugin/iso19139/templates/raster.xml
XSL file: /data/dev/gn/iso19115-3/src/main/plugin/iso19115-3/convert/ISO19139/fromISO19139.xsl
Engine name: Saxon-HE 9.5.1.3
Severity: fatal
Description: XPTY0004: A sequence of more than one item is not allowed as the first argument of string-length() ("grid", "vector")
Start location: 69:0
URL: http://www.w3.org/TR/xpath20/#ERRXPTY0004
``` 


Expected:
```

         <mri:spatialRepresentationType>
            <mcc:MD_SpatialRepresentationTypeCode codeList="codeListLocation#MD_SpatialRepresentationTypeCode"
                                                  codeListValue="grid">grid</mcc:MD_SpatialRepresentationTypeCode>
         </mri:spatialRepresentationType>
         <mri:spatialRepresentationType>
            <mcc:MD_SpatialRepresentationTypeCode codeList="codeListLocation#MD_SpatialRepresentationTypeCode"
                                                  codeListValue="vector">vector</mcc:MD_SpatialRepresentationTypeCode>
         </mri:spatialRepresentationType>
```

**Comment by fxprunayre** [comment]
Fixed in 3.4.3 https://github.com/metadata101/iso19115-3/commit/21def720ea0701f7eec9af05d130092f1428ddbe and 3.4.x https://github.com/metadata101/iso19115-3/commit/25ca92607a08e82b82aa67c35dd620d0d8c0bd62

Let me know if you still have this type of issue and if not, please close this ticket. Thanks.
